From cab6f2824d122fb25d05e2f4351b9b7ebf78095f Mon Sep 17 00:00:00 2001
From: Adil Hanney <adilhanney@disroot.org>
Date: Sun, 24 May 2026 12:41:11 +0100
Subject: [PATCH] Disable prebuilt pdfium

---
 packages/pdfium_dart/hook/build.dart | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/packages/pdfium_dart/hook/build.dart b/packages/pdfium_dart/hook/build.dart
index 24284e7..f152b2a 100644
--- a/packages/pdfium_dart/hook/build.dart
+++ b/packages/pdfium_dart/hook/build.dart
@@ -44,6 +44,9 @@ Future<void> _downloadPdfium({
   final output = File.fromUri(outputFile);
   if (await output.exists()) return;
 
+  throw 'Cannot download prebuilt pdfium on F-Droid. '
+      'Please build it beforehand.';
+
   final archiveUri = Uri.parse(
     'https://github.com/bblanchon/pdfium-binaries/releases/download/'
     '$pdfiumRelease/pdfium-${target.archivePlatform}-${target.archiveArch}.tgz',
-- 
2.54.0

